python/lsst/meas/algorithms/gaussianPsfFactory.py

```diff
@@ -23,7 +23,7 @@
 import math
 
 from lsst.pex.config import Config, Field, ConfigurableField
-from .algorithmsLib import SingleGaussianPsf, DoubleGaussianPsf
+from . import SingleGaussianPsf, DoubleGaussianPsf
```

Comment:
Can you please clarify what you are asking for? This looks like a standard import to me.